src/extensions/default/CodeFolding/foldhelpers/latex-fold.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Fold latex code regions
```

Comment:
I wonder if a better approach would be to make this more extensible -- an important goal of the LanguageManager work is to make it so any feature in Brackets can be enabled for additional languages just by installing an extension.  Latex and the custom regions feature both seem like potential candidates to move out into extensions.

Custom regions also adds a menu item that I suspect few users will understand, and that's concerning to me.  Since refactoring this to be more extensible is probably a post-1.3 thing, I wonder if it'd be better to just disable the custom regions feature for now?  Or have the menu item hidden by default, and accessible only by setting a preference?